services/credential-server-ui/package.json

```diff
@@ -29,7 +29,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "chmod +x ./env.sh && ./env.sh && cp env-config.js ./public/ && react-scripts start",
```

Comment:
This seems quite convoluted but I guess it's because we are using `react-scripts` and not `webpack` or `vite` like in other projects. Since this is just a demo/testing UI, I'm tempted to just merge this though, what do you think @rcmorano ?

Reply:
I'd say we don't need it for the dev server as react will pass any `REACT_APP_*` environment variable to the process.

Also, this method would require as wether a) add an `.env` file at build time so the entrypoint can parse it (the anti-pattern we actually want to avoid) or b) mount an `.env` file to the live container, which is not the native way to pass environment variables.

I suggest we mimick the react dev server by parsing any env var matching the `^REACT_APP_*` pattern and make it available in the `env-config.js`.